pandas_datareader/enigma.py

```diff
@@ -32,7 +32,8 @@ def __init__(self,
                  api_key=None,
                  retry_count=5,
                  pause=.75,
-                 session=None):
+                 session=None,
+                 base_url="https://public.enigma.com/api"):
```

Comment:
Needs something added to the docstring.  Ideally, the docstring would be rewritten to include all parameters.

Reply:
@bashtage Added a `Parameters` section to the `EnigmaReader` docstring. Let me know what you think.